dev/push-setup/push-setup

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+WORKSPACE_DIR=.chef
+
+setup_workspace() {
+    echo "Setting up workspace."
```

Comment:
Here a little PRO tip for the output.. If you use the first `echo` with the option `-n` it will not do the carrier return.. then the next `echo` will be printing in the same line.. You could do something like:
```
setup_workspace() {
    echo -n "Setting up workspace: "
    mkdir -p $WORKSPACE_DIR
    echo "Done."
}
```

And it should looks like:
```
Setting up workspace: Done
```

You can apply this to the rest of the script.. (if you want)